signature/internal/base/envelope.go

```diff
@@ -156,7 +156,7 @@ func validateSignerInfo(info *signature.SignerInfo) error {
 
 	return validateCertificateChain(
 		info.CertificateChain,
-		signingTime,
+		time.Time{},
```

Comment:
Looks like the code you mentioned in [notation-go](https://github.com/notaryproject/notation-go/blob/main/verification/verifier.go#L183-L187) does not check the SigningTime? It's for "verify x509 trust store based authenticity". I might be missing it, could you point to the singingTime related logic in notation-go?

Reply:
[Here](https://github.com/notaryproject/notation-go/blob/2bcfd343f974938e8eeb76e233bc2526e0c36061/verification/verifier_helpers.go#L156-L170) we are performing verification.